pkg/features/kube_features.go

```diff
@@ -897,7 +898,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DisableKubeletCloudCredentialProviders: {Default: false, PreRelease: featuregate.Alpha},
 
-	DownwardAPIHugePages: {Default: true, PreRelease: featuregate.Beta}, // on by default in 1.22
+	DownwardAPIHugePages: {Default: true, PreRelease: featuregate.GA}, // on by default in 1.22
```

Comment:
set `LockToDefault:true`, add a comment to remove in 1.29, and drop code that checks if this gate is enabled

Reply:
Yep, done :+1: